content/en/agent/cluster_agent/external_metrics.md

```diff
@@ -135,8 +163,29 @@ For autoscaling to work correctly, the query must follow these rules:
 **Note**: While the query is arbitrary, the start and end times are still set at `Now() - 5 minutes` and `Now()`
 
 ### Set up Datadog Cluster Agent to use DatadogMetric
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+To activate usage of the `DatdogMetric` CRD update your [datadog-values.yaml][1] Helm configuration to set `clusterAgent.metricsProvider.useDatadogMetrics` to `true`.  Then redeploy your Datadog Helm chart:
+
+  ```yaml
+  clusterAgent:
+    enabled: true
+    metricsProvider:
+      enabled: true
+      # clusterAgent.metricsProvider.useDatadogMetrics
+      # Enable usage of DatadogMetric CRD to autoscale on arbitrary Datadog queries
+      useDatadogMetrics: true
+  ```
 
-To activate usage of `DatadogMetric` CRD, follow these extra steps:
+**Note:** This will automatically attempt to install the `DatadogMetric` CRD. If that CRD already exists prior to the initial Helm instalation it may conflict.
```

Comment:
```suggestion
**Note:** This will automatically attempts to install the `DatadogMetric` CRD. If that CRD already exists prior to the initial Helm installation, it may conflict.
```